documentation/book/proc-accessing-kafka-using-nodeports.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring-kafka-listeners.adoc
+
+[id='proc-accessing-kafka-using-nodeports-{context}']
+= Accessing Kafka using node ports routes
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Deploy Kafka cluster with external listener enabled and configured to the type `nodeport`.
```

Comment:
Deploy a Kafka cluster with an external listener enabled and configured with the type `nodeport`.